flash/core/schedulers.py

```diff
@@ -0,0 +1,10 @@
+from flash.core.registry import FlashRegistry
+from flash.utils.imports import _TRANSFORMERS_AVAILABLE
+
+_SCHEDULER_REGISTRY = FlashRegistry("scheduler")
+
+if _TRANSFORMERS_AVAILABLE:
+    from transformers import optimization
```

Comment:
Why only add transformers optimizers here? Maybe also add `torch.optim` optimizers?

Reply:
transformers already provide lambda lr. Users can provide their own scheduler if needed. Registry should be more custom things IMO.

Reply:
Ok, I'm a little confused. The user can just provide a scheduler from transformers too right?